### ISSUE TITLE
Multicluster dashboard empty?

### ISSUE THREAD

**Issue by grampelberg** [linkerd/linkerd2#4555]
## What is the issue?

<img width="1194" alt="Screenshot 2020-06-04 11 26 00" src="https://user-images.githubusercontent.com/47992/83796659-3e3ffa80-a656-11ea-9deb-2090e7684d14.png">

Some other output:

```bash
$ linkerd check --multicluster
linkerd-multicluster
--------------------
√ service mirror controller is running
√ service mirror controller ClusterRoles exist
√ service mirror controller ClusterRoleBindings exist
√ service mirror controller Roles exist
√ service mirror controller RoleBindings exist
√ service mirror controller ServiceAccounts exist
√ service mirror controller has required permissions
√ service mirror controller can access target clusters
√ all target cluster gateways are alive
	* cluster: [], gateway: [linkerd-multicluster/linkerd-gateway]
√ clusters share trust anchors
√ multicluster daisy chaining is avoided
$ linkerd --context=west multicluster gateways
CLUSTER  NAMESPACE             NAME             ALIVE    NUM_SVC  LATENCY_P50  LATENCY_P95  LATENCY_P99
         linkerd-multicluster  linkerd-gateway  True           0          3ms          4ms          4ms
```

Maybe it is because my cluster name is empty? Note, this is on edge-20.6.1

**Comment by adleong** [comment]
I think this is probably a duplicate of #4553 and will be fixed by #4554

**Comment by grampelberg** [comment]
That's what I get for not looking at issues first. Thanks @adleong !
